test/ginkgo-e2e/querylogs/querylogs_test.go

```diff
@@ -0,0 +1,51 @@
+package querylogs_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"docker-provider/test/utils"
+)
+
+var _ = Describe("When querying the logs for the table", func() {
+	DescribeTable("All tables should have logs",
+		func(table string) {
+			query := table + " | where TimeGenerated > ago(15m) | summarize count()"
+			err := utils.QueryLogsForCount(LogsClient, AKSResourceId, query, false)
+			Expect(err).NotTo(HaveOccurred())
+		},
+		// Check how to chose containerlog table
+		Entry("ContainerLog", "ContainerLog"),
```

Comment:
ContainerLogV2 can be configured either through config map or DCR.  There are multiple ways to handle. May be best option to check the count is greater than 0 either for ContainerLog or ContainerLogV2.